### ISSUE TITLE
Version Converter fails with Assertion assertInputsAvailable

### ISSUE THREAD

**Issue by paulbauriegel** [onnx/onnx#2191]
#### System Environment 
- OS: `Ubuntu 19.04`
- ONNX Version: `1.5.0`
#### Problem
I was trying to convert the version of a ONNX Model and I'm getting always the same exception, no matter what version I try.
```
---------------------------------------------------------------------------
RuntimeError                              Traceback (most recent call last)
<ipython-input-151-79957b488afe> in <module>
      2 inferred_model = shape_inference.infer_shapes(model)
      3 print("model_lcnn_29v2.onnx has IR version", inferred_model.ir_version)
----> 4 converted_model = version_converter.convert_version(inferred_model, 3)

/opt/conda/lib/python3.7/site-packages/onnx/version_converter.py in convert_version(model, target_version)
    164         raise ValueError('VersionConverter only accepts int as target_version, incorrect type: {}'.format(type(target_version)))
    165     model_str = model.SerializeToString()
--> 166     converted_model_str = C.convert_version(model_str, target_version)
    167     return onnx.load_from_string(converted_model_str)

RuntimeError: /tmp/pip-req-build-g_wgbzen/onnx/version_converter/helper.cc:62: assertInputsAvailable: Assertion `inputs[i]->has_sizes()` failed: Shape of input 2 is not available.
```
I've accounted the problem with a ONNX model with IR Version 4 which I exported from PyTorch. [Link to the Model](https://drive.google.com/open?id=1tOKjcP3iJg0vDxq4valpcxliIt5-rm4T) using that code snipped:
```python
model = onnx.load("model_lcnn_29v2.onnx")
print("model_lcnn_29v2.onnx has IR version", model.ir_version)
converted_model = version_converter.convert_version(model, 3)
```
I've benchmarked the inference of the model using the ONNX runtime and the results have been fine so I believe the model is valid.

**Comment by ebarsoum** [comment]
The just released PyTorch 1.2, support exporting to different OPSet version.

**Comment by sathiiii** [comment]
> 

Were you able to resolve this? @paulbauriegel 

I'm facing a similar issue, but can't find the reason behind that. Is it because of unsupported ONNX Adapters?
